config/locales/views/users/nb.yml

```
@@ -7,6 +7,9 @@ nb:
       email_change_initiated: Vennligst sjekk din nye e-postadresse for bekreftelsesinstruksjoner.
       success: Profilen din har blitt oppdatert.
-    reset:
-      success: Kontoen din har blitt tilbakestilt. Data vil bli slettet i bakgrunnen om en stund.
-      unauthorized: Du er ikke autorisert til å utføre denne handlingen.
\ No newline at end of file
+    reset:
+      success: Kontoen din har blitt tilbakestilt. Data vil bli slettet i bakgrunnen om en stund.
+      unauthorized: Du er ikke autorisert til å utføre denne handlingen.
+    reset_with_sample_data:
+      success: Kontoen din har blitt tilbakestilt og demodata forberedes. Du vil snart se eksempeldata.
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)